### ISSUE TITLE
lint --fail-severity parameters missing in documentation

### ISSUE THREAD

**Issue by mazumba-nx** [daveshanley/vacuum#377]
I am using the docker container `dshanley/vacuum:v0.5.1` in a github workflow to check for OpenAPI errors.
The command is `vacuum lint apidoc.yaml --no-style --fail-severity warning` and should return Error 1 when warnings or errors are thrown.

When using the lint command with a file having errors and warnings, the command only exits with Error 1 when either
- use `--fail-severity error` (or using the default)
- use `--fail-severity info`

When using `--fail-severity warning`, the command displays the errors and warnings but still returns successfully.
Here is an example yaml with an error and a couple warnings.

```
openapi: 3.0.0
info:
  title: 'An Example API'
  description: 'Provide an example for an issue'
  version: 1.0.0
servers:
  -
    url: /api
paths:
  '/api/foo/{id}/bar':
    get:
      tags:
        - Foo
      description: 'Lists orders'
      parameters:
        -
          name: id
          in: path
          required: true
          schema:
            type: string
      responses:
        '200':
          description: 'The list of Bars'
          content:
            application/json:
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/Bar'
components:
  schemas:
    Bar:
      properties:
        id:
          type: string
      type: object
```

I think there is a `warning` key missing somewhere in the code.

Hope this helps! Thank you!

**Comment by mazumba-nx** [comment]
After thinking "maybe I can resolve this and create a PR" I found, that the correct parameter is `warn` and not `warning`. So no Bug here!

This is not clear from the documentation though. https://quobix.com/vacuum/commands/lint/

Could you include the three possibilities `error`, `warn` and `info` to the list?

**Comment by ajoga** [comment]
+1 :) this echoes https://github.com/daveshanley/vacuum/issues/367#issuecomment-1816071341

**Comment by daveshanley** [comment]
A new `version` command is available in `v0.6.0`

https://quobix.com/vacuum/commands/version/

Also the docs for the `--fail-severity` flag has been properly documented. 

https://quobix.com/vacuum/commands/lint/#fail-severity
